cmd/faucet/faucet.go

```diff
@@ -1,1244 +0,0 @@
-// Copyright 2017 The go-ethereum Authors
```

Comment:
@meowsbits do we want to keep and maintain faucet as you were using it?

Reply:
I was using it for Mordor. If we have, or there are, other ways of distributing tokens then I'm fine abandoning or archiving it.

Reply:
@diega I think you also have a faucet. Which backend does it use?

Reply:
It's just a CF worker that gives away 1 mETC to any address with less than 2 mETC that I fill from the mining rewards of the Besu node that I maintain